mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -45,7 +46,8 @@ private[regression] trait GeneralizedLinearRegressionBase extends PredictorParam
    * to be used in the model.
    * Supported options: "gaussian", "binomial", "poisson" and "gamma".
    * Default is "gaussian".
-   * @group param
+    *
```

Comment:
This may be what you want:

1. "Preferences" -> "Editor" -> "Code Style" -> "Scala" 
2. Select "ScalaDoc" tab
3. Uncheck "Use scaladoc indent for leading asterisk"